gui/src/app/SPAnalysis/SPAnalysisQueryLoading.ts

```diff
@@ -76,8 +81,19 @@ const deepCopy = (obj: any) => {
 }
 
 export const fetchRemoteAnalysis = async (query: QueryParams) => {
-    // any special 'project' query could be loaded here at the top
-    const data: SPAnalysisDataModel = deepCopy(initialDataModel)
+    const projectUri = query.project
+
+    let data: SPAnalysisDataModel = deepCopy(initialDataModel);
+    if (projectUri) {
+        if (projectUri.startsWith('https://gist.github.com/')) {
+            const contentLoadedFromGist = await loadFilesFromGist(projectUri);
+            data = loadFromProjectFiles(data, mapFileContentsToModel(contentLoadedFromGist.files), false);
+            data.meta.title = contentLoadedFromGist.description;
```

Comment:
Should we `return` right after this?

Reply:
Depends on if we want to support project plus other overrides. If we choose not to support overrides, then we need to provide an alert if they are not being respected. But I would vote for allowing overrides, parameters in addition to project, as it is here.

Reply:
Overrides seem reasonable in this case.